data/settings.js

```diff
@@ -1125,14 +1277,14 @@ characters \`(strlen)\`, e.g. \`'*******'\`, or a textual description
 \`(text)\`, e.g. \`'Spam'\` or \`'Not Spam'\`.`
 	},
 
```

Comment:
sieve_spamtest_status_type:
 * It's not really ENUM type in the code right now. I think it can be changed?
 * The \`(foo)\` quoting is confusing. I think it should be (\`foo\`)
 * Add a link to sieve_spamtest_text_value setting

Same for sieve_virustest

Reply:
fixed